webapps-schema/src/main/resources/schema/elasticsearch/create/template/tasklist-task.json

```diff
@@ -0,0 +1,123 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "join": {
+        "type": "join",
+        "eager_global_ordinals": true,
+        "relations": {
+          "task": [
+            "taskVariable"
+          ],
+          "process": [
+            "processVariable",
+            "task"
+          ]
+        }
+      },
+      "flowNodeBpmnId": {
+        "type": "keyword"
+      },
+      "flowNodeInstanceId": {
+        "type": "keyword"
+      },
+      "partitionId": {
+        "type": "integer"
+      },
+      "completionTime": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "processInstanceId": {
+        "type": "keyword"
+      },
+      "id": {
+        "type": "keyword"
+      },
+      "position": {
+        "type": "long"
+      },
+      "state": {
+        "type": "keyword"
+      },
+      "key": {
+        "type": "long"
+      },
+      "creationTime": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "bpmnProcessId": {
+        "type": "keyword"
+      },
+      "processDefinitionId": {
+        "type": "keyword"
+      },
+      "assignee": {
+        "type": "keyword"
+      },
+      "candidateGroups": {
+        "type": "keyword"
+      },
+      "candidateUsers": {
+        "type": "keyword"
+      },
+      "formKey": {
+        "type": "keyword"
+      },
+      "formId": {
+        "type": "keyword"
+      },
+      "formVersion": {
+        "type": "long"
+      },
+      "isFormEmbedded": {
+        "type": "boolean"
+      },
+      "followUpDate": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "dueDate": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "tenantId": {
+        "type": "keyword"
+      },
+      "implementation": {
+        "type": "keyword"
+      },
+      "externalFormReference": {
+        "type": "keyword"
+      },
+      "processDefinitionVersion": {
+        "type": "integer"
+      },
+      "customHeaders": {
+        "type": "object",
+        "dynamic": true
+      },
+      "priority": {
+        "type": "integer"
+      },
+      "name": {
+        "type": "keyword"
+      },
+      "value": {
+        "type": "keyword",
+        "ignore_above": 8191
+      },
+      "fullValue": {
+        "type": "keyword",
+        "ignore_above": 8191
+      },
+      "scopeKey": {
+        "type": "keyword"
+      },
+      "isTruncated": {
+        "type": "boolean"
+      }
```

Comment:
❓ What about 

```

      "isPreview": {
        "type": "boolean"
      },
      "dataType": {
        "type": "keyword"
      }
```

Reply:
The `isPreview` field is being replaced by `isTruncated` for consistency with the new REST API format.
The `dataType` will not be needed.